chain/events/filter/event.go

```diff
@@ -100,9 +100,18 @@ func (f *EventFilter) CollectEvents(ctx context.Context, te *TipSetEvents, rever
 				continue
 			}
 
+			decodedEntries := make([]types.EventEntry, len(ev.Entries))
+			for i, entry := range ev.Entries {
+				decodedEntries[i] = types.EventEntry{
+					Flags: entry.Flags,
+					Key:   entry.Key,
+					Value: decodeLogBytes(entry.Value),
+				}
+			}
```

Comment:
Doesn't this go in the opposite direction than Steb pointed out in https://github.com/filecoin-project/ref-fvm/issues/1345 (1)?